src/Definition/ConfigProcessor/AclConfigProcessor.php

```diff
@@ -47,21 +47,15 @@ public static function acl(array $fields, AccessResolver $accessResolver, callab
         return $fields;
     }
 
-    public function process(LazyConfig $lazyConfig): LazyConfig
+    public function process(array &$config): void
```

Comment:
Passing config array by reference to avoid copy of an array for performance reasons.